### PR TITLE
Migrate GitVersionControlSpec

### PR DESCRIPTION
**Pull request by mlopatkin**

### CODE REVIEW

**Comment by mlopatkin** [comment]
@bot-gradle test this

**Comment by mlopatkin** [comment]
@bot-gradle test this

File: platforms/native/language-native/src/main/java/org/gradle/swiftpm/plugins/SwiftPackageManagerExportPlugin.java [hunk 1]
```diff
@@ -214,35 +215,36 @@ private void collectDependencies(Configuration configuration, Collection<Depende
                         throw new InvalidUserDataException(String.format("Cannot determine the Git URL for dependency on %s:%s.", dependency.getGroup(), dependency.getName()));
                     }
                     GitVersionControlSpec gitSpec = (GitVersionControlSpec) vcsSpec;
-                    dependencies.add(toSwiftPmDependency(externalDependency, gitSpec));
+                    gitSpec.getUrl().finalizeValue();
+                    dependencies.add(toSwiftPmDependency(externalDependency, gitSpec.getUrl().get()));
```
Comment (mlopatkin) [inline platforms/native/language-native/src/main/java/org/gradle/swiftpm/plugins/SwiftPackageManagerExportPlugin.java:219]: 💭  This place is being called lazily, so it is safe to resolve the property. We don't carry dependencies, but you cannot really have dependencies for your git url.

**Comment by asodja** [comment]
@bot-gradle test RFN

**Pull request merged**